src/gam.py

```diff
@@ -8330,26 +8330,23 @@ def doWhatIs():
     sys.stderr.write(u'%s is a group alias\n\n' % email)
     doGetAliasInfo(alias_email=email)
 
-def convertSKU2ProductId(res, sku, customerId):
-  subscriptionId = None
-  results = callGAPI(res.subscriptions(), u'list', customerId=customerId)
-  for subscription in results[u'subscriptions']:
-    if sku == subscription[u'skuId']:
-      subscriptionId = subscription[u'subscriptionId']
-      break
-  if not subscriptionId:
-    print u'ERROR: could not find subscription for customer %s and SKU %s' % (customerId, sku)
-    sys.exit(3)
-  return subscriptionId
+def getCustomerSubscription(res):
+  customerId = sys.argv[3]
+  skuId = sys.argv[4]
```

Comment:
I don't like these utility functions depending on sys.argv order, it means they can't be used later if argument order is different. I think convertSKU2ProductId() is pretty much fine as is though the argument cleanup in _getResoldCustomerAttr() is good.